kleros/escrow
# fix: handle missing amount and extra data meta evidence properties

# Description

This PR fixes the handling of missing meta evidence properties, `amount` and `extraData`.

# Rationale

Transactions missing these properties should still display in the UI.

# How To Test This?

Create an escrow transaction with no `amount` and or `extraData` and verify that the UI behaves as expected.
